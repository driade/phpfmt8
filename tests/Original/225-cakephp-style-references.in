<?php
//passes:Default,CakePHPStyle
//version:8
$a = &$b;
