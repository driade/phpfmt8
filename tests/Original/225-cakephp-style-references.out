<?php
//passes:Default,CakePHPStyle
//version:8.2
$a = &$b;
